### ISSUE TITLE
Upload stable release archive

### ISSUE THREAD

**Issue by jondo2010** [cameron314/readerwriterqueue#146]
Hi there,

I'm working on adding `readerwriterqueue` as a Bazel Module to the [Bazel Central Repository](https://registry.bazel.build/). My PR is here: https://github.com/bazelbuild/bazel-central-registry/pull/1088

*Bazel Central Repository* requires that packages point to a "Release Archive", not just a source-code archive on Github. This is because the source-code archives are automatically generated on-demand by Github, and are not guaranteed to be hash-stable.

More details here: https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/

tldr; Please upload a static release archive of the source code for the latest release to GH

**Comment by a4z** [comment]
see also #144
